pkg/gofr/datasource/clickhouse/clickhouse_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build exclude
+// +build exclude
```

Comment:
Not exactly related to this PR, but this syntax is outdated for a while I think.

https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

@vipul-rawat @Umang01-hash Maybe an new issue/another PR could be created

What do you think?

Reply:
I've opened 

https://github.com/gofr-dev/gofr/issues/1175

Reply:
Thank you for opening the issue. Marking this as resolved. Will remove it altogether when addressing the issue